rudder-lang/src/parser/tests.rs

```diff
@@ -1091,11 +1135,11 @@ fn test_pstatement() {
                 "case".into(),
                 vec![
                     (
-                        map_res(penum_expression, "ubuntu").unwrap().1,
+                        map_res(penum_expression, "ubuntu ").unwrap().1,
```

Comment:
is this really normal?

Reply:
yes, parsers are eager on whitespaces, so the "source" field always contains final spaces

-> they are need when comparing the result of 2 parsing